dps-web-experience-sample/Views/Shared/_DocsNav.cshtml

```diff
@@ -126,9 +126,9 @@
 
                 <li id="tutorial-ionic"><a href="~/en-us/docs/tutorial-ionic/">Get started with Ionic</a></li>
 
-                <li id="O365_Files"><a href="~/en-us/docs/o365_files/">O365 Files API</a></li>
```

Comment:
<li> id needs to be the same as the document id. So please update the id as well.